tests/document/instance.py

```diff
@@ -1552,9 +1593,9 @@ class ParentModel(Document):
                 EmbeddedChildModel)
 
         emb = EmbeddedChildModel(id={'1': [1]})
-        ParentModel(children=emb)._get_changed_fields()
+        ParentModel(child=emb)._get_changed_fields()
 
-    def test__get_changed_fields_same_ids_reference_field_does_not_enters_infinite_loop(self):
+    def test__get_changed_fields_same_ids_reference_field_does_not_enters_infinite_loop_full_caseEmailUser(self):
```

Comment:
Just curious, what was the reason for this rename? That test name's quite something btw 😅

Reply:
I have a pre-commit hook that runs flake8 and it highlighted that there were 2 methodswith that name in this file (which means only 1 of them run) so I had to come up with a different name for one of the two. The "EmailUser" there is a copy/paste mistake though, I'll remove that immediatly